src/Evolution/Systems/Cce/KleinGordonSystem.hpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
```

Comment:
Please add in all the `#includes` you are missing: tmpl, cce tags, Variables tag

Reply:
Ping on this. You need to add the includes to the KG system as well.